force_gromacs/gromacs_plugin.py

```diff
@@ -39,6 +39,5 @@ def get_factory_classes(self):
         return [
             FragmentFactory,
             MoleculeFactory,
-            SimulationFactory,
```

Comment:
I've removed the `SimulationFactory` from the plugin, as the `SimulationDataSource` contains methods that need to be implemented (and so cannot be run 'out-of-the-box')